### PR TITLE
Added File Specific Documentation

### LINKED ISSUE #2: [Docs] - Add documentation for the codebase
It's always great to have documentation about the codebase, as it would help new developers and contributors to the project!

It would be great to have a `docs/` directory and having documentation neatly structured for the entire repository!

These are the primary files for which documentation is required:
- `theatre_cli/movie.py`
- `theatre_cli/player.py`
- `theatre_cli/config.py`
- `scripts/theatre-cli`

How to structure the `docs/` directory? -> It's upto you, just keep it clean!

Also, in addition to this, a brief introduction to rest of the files and the file structure, would be greatly appreciated!

### PR DESCRIPTION
**Pull request by nayanmapara**
Fixes #2

### CODE REVIEW

**Comment by nayanmapara** [comment]
@virajsazzala Let me know if you would like me to make any changes

**Comment by virajsazzala** [comment]
@nayanmapara I'm sorry for the delay, I'm stuck with some exam, I'll get back to you, by 17:00IST 12OCT.

File: docs/scripts.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+To use the script, execute it from the command line as follows:
+
+```bash
+./theatre-cli
+```
+
+## Command-line Options:
+
+* `--update-dir PATH`: Updates the movie directory to the specified path.
```
Comment (virajsazzala) [inline docs/scripts.md]: Wouldn't it be better to use angular braces for variables, like `<VARNAME>` ?

```suggestion
* `--update-dir <PATH>`: Updates the movie directory to the specified path.
```

File: docs/theatre_cli/config.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# `theatre_cli/config.py`
+
+The `config.py` module is responsible for managing configuration settings for your Theater CLI application. It handles loading and saving user settings, such as the movie directory and first-time setup.
+
+## Functions:
+1. `load_config()`
+    * Description: Loads the configuration from a file named "config.json." It returns the configuration data as a list of dictionaries.
```
Comment (virajsazzala) [inline docs/theatre_cli/config.md]: In the entire documentation, I think it's better to bold labels like `Description`, `Usage` etc..

File: docs/theatre_cli/config.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    * Usage:
+        ```python
```
Comment (virajsazzala) [inline docs/theatre_cli/config.md]: In the entire documentation, can you give one line space before writing a code block for usage?

```suggestion
    * Usage:
    
        ```python
```

**Review by virajsazzala — CHANGES_REQUESTED** [review]

**Comment by nayanmapara** [comment]
I have implemented all the changes have a look @virajsazzala

**Comment by virajsazzala** [comment]
@nayanmapara Perfect! Thanks for contributing to theatre-cli!

**Comment by virajsazzala** [comment]
LGTM!

**Review by virajsazzala — APPROVED** [review]

**Pull request merged**